src/App.js

```diff
@@ -8,16 +8,29 @@ import LandingPage from "./pages/LandingPage/LandingPage";
 import NotesPage from "./pages/NotesPage/NotesPage";
 import ProfilePage from "./pages/ProfilePage/ProfilePage";
 import SocialPage from "./pages/SocialPage/SocialPage";
-
+// TODO: set state for rendering dashboard and landing page with boolean value 
 function App() {
+  const [loggedIn, setLoggedIn] = useState(false);
+ const handleClick = () => {
+  if (loggedIn) {
+    setLoggedIn(false)
+    console.log(document.querySelector('#loginBtn'))
+
+    document.querySelector('#loginBtn').textContent = "login"
+  } else {
+    setLoggedIn(true)
+    document.querySelector('#loginBtn').textContent = "logout"}
+ }
   return (
   <div className="appContainer">
             {/* <Link to="/dashboard">Dashboard</Link>  */}
     <header>
       <Header />
     </header>
+    <button id='loginBtn' onClick={handleClick}>login</button>
     <main>
-      <LandingPage />    
+      {!loggedIn && <LandingPage />}
```

Comment:
I think we can combine lines 32-33 into a ternary:
!loggedIn ? <LandingPage /> : <DashboardPage />;